### PR TITLE
feat: generate ts protos

### PR DESCRIPTION
**Pull request by hspitzley-czi**
cd into `api/proto/hapi/` and run `yarn install` then run `./generate.sh` to generate protos

### CODE REVIEW

File: api/proto/hapi/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "hapi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (hspitzley-czi) [inline api/proto/hapi/package.json:10]:
```suggestion
  "name": "happy-api-protos",
  "version": "1.0.0",
  "description": "Protobufs for happy api"
```

File: api/proto/hapi/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "happy-api-protos",
+  "version": "1.0.0",
+  "description": "Protobufs for happy api"
```
Comment (hspitzley-czi) [inline api/proto/hapi/package.json:4]:
```suggestion
  "description": "Protobufs for happy api",
```

File: api/proto/hapi/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "happy-api-protos",
+  "version": "1.0.0",
```
Comment (hspitzley-czi) [inline api/proto/hapi/package.json:3]:
```suggestion
  "version": "0.1.0",
```

**Comment by dspadea** [comment]
I've incorporated this manually into the next release of hapi protos. Thanks! 

https://github.com/chanzuckerberg/happy/pull/2389

**Pull request closed without merging**